source/tyr/migrations/versions/55bf63d1c18_add_fields_enable_free_on_users.py

```diff
@@ -0,0 +1,28 @@
+"""add fields "enable_free_instances" and "is_super_user" to Users
+
+Revision ID: 55bf63d1c18
+Revises: 2a77596bdf40
+Create Date: 2015-04-13 11:51:00.480619
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '55bf63d1c18'
+down_revision = '2a77596bdf40'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user', sa.Column('enable_free_instances', sa.Boolean(), nullable=False, server_default='True'))
+    op.add_column('user', sa.Column('is_super_user', sa.Boolean(), nullable=False, server_default='False'))
+    ### end Alembic commands ###
+
+
+def downgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('user', 'is_super_user')
+    op.drop_column('user', 'enable_free_instances')
```

Comment:
Better to have 3 kinds of users: {with_free_instances, without_free_instances, super_user} ?